packages/typespec-vscode/package.json

```diff
@@ -232,7 +247,7 @@
     "copy-tmlanguage": "node scripts/copy-tmlanguage.js",
     "generate-language-configuration": "node scripts/generate-language-configuration.js",
     "generate-third-party-notices": "typespec-build-tool generate-third-party-notices",
-    "package-vsix": "vsce package",
+    "package-vsix": "vsce package --no-dependencies",
```

Comment:
do we still need the --no-dependencies?

Reply:
I tested on local. It's not required. Rolled back to current command.